types/ion-rangeslider/index.d.ts

```diff
@@ -0,0 +1,81 @@
+// Type definitions for ion-rangeslider 2.2
+// Project: https://github.com/IonDen/ion.rangeSlider/
+// Definitions by: Karel van de Plassche <https://github.com/Karel-van-de-Plassche>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+// API documentation: http://ionden.com/a/plugins/ion.rangeSlider/en.html
+// Parsed using https://github.com/Karel-van-de-Plassche/ion-rangeslider-parser.git
+// Based on global-modifying module template file
+
+declare global {
+    interface JQuery {
+        destroy(): void;
+        ionRangeSlider(options?: IonRangeSliderOptions): JQuery;
+        reset(): void;
+        update(option: IonRangeSliderOptions): void;
+    }
+}
+
+export interface IonRangeSliderOptions {
+    type?: string;    // Choose slider type, could be `single` - for one handle, or `double` for two handles [Default: single]
```

Comment:
can you alph these...? kind of hard to compare whats changed

Reply:
I'm not sure what you mean. For this file I have:
- Added `declare global { }` enclosure
- Updated the comments in the file header
- Added comment strings after each argument (which I prefer, but I'm not sure about the TS crowd)

Reply:
Git didn't detect the rename because the file changed too much, presumably. Changes look OK to me.

Reply:
its easier to search and browse through the list... when there's no easy functional grouping alph is preferred IMO :)

Reply:
Fair point. If you want to open a PR to alphabetize these I would approve it.

Reply:
Right now I conserved the order from the documentation site of the library (http://ionden.com/a/plugins/ion.rangeSlider/en.html) which has some loose ordering based on functionality. If alphabetical is preferred, I don't mind opening a new pull request

Reply:
Right now I conserved the order from the documentation site of the library (http://ionden.com/a/plugins/ion.rangeSlider/en.html) which has some loose ordering based on functionality. If alphabetical is preferred, I don't mind opening a new pull request